### PR TITLE
pulseeffects: update to 4.8.4

### PR DESCRIPTION
**Pull request by ndowens**
Looked at https://github.com/void-linux/void-packages/pull/2566 and I do not get the issue reported, the following is what I get and pulseeffects launches fine.
```
PULSEEFFECTS_DEBUG=1 pulseeffects

(pulseeffects:31375): Gtk-WARNING **: 19:05:36.812: Theme parsing error: gtk.css:2:33: Failed to import: Error opening file /home/ndowens/.config/gtk-3.0/window_decorations.css: No such file or directory

(pulseeffects:31375): pulseeffects-WARNING **: 19:05:36.867: compressor plugin was not found!

(pulseeffects:31375): pulseeffects-WARNING **: 19:05:36.869: equalizer plugin was not found!

(pulseeffects:31375): pulseeffects-WARNING **: 19:05:36.878: maximizer plugin was not found!

(pulseeffects:31375): pulseeffects-WARNING **: 19:05:36.882: loudness plugin was not found!

(pulseeffects:31375): pulseeffects-WARNING **: 19:05:36.884: pitch plugin was not found!

(pulseeffects:31375): pulseeffects-WARNING **: 19:05:36.890: convolver plugin was not found!

(pulseeffects:31375): pulseeffects-WARNING **: 19:05:36.890: crystalizer plugin was not found!

(pulseeffects:31375): pulseeffects-WARNING **: 19:05:36.892: delay plugin was not found!

(pulseeffects:31375): pulseeffects-WARNING **: 19:05:36.897: compressor plugin was not found!

(pulseeffects:31375): pulseeffects-WARNING **: 19:05:36.897: equalizer plugin was not found!

(pulseeffects:31375): pulseeffects-WARNING **: 19:05:36.899: pitch plugin was not found!

(pulseeffects:31375): pulseeffects-WARNING **: 19:05:36.905: maximizer plugin was not found!

```

### CODE REVIEW

File: srcpkgs/pulseeffects/template [hunk 1]
```diff
@@ -1,18 +1,20 @@
 # Template file for 'pulseeffects'
 pkgname=pulseeffects
 reverts="4.3.4_1"
```
Comment (ericonr) [inline srcpkgs/pulseeffects/template:3]: This needs to remove the revert.

**Comment by ericonr** [comment]
Is this still good after the pulseaudio update?

**Comment by ndowens** [comment]
> Is this still good after the pulseaudio update?

It was working with version when I updated it but if version has changed I am unsure; I no longer use pulseaudio

**Comment by ericonr** [comment]
Can you update to 4.8.4? At least there won't be new releases in this branch: https://github.com/wwmm/pulseeffects/blob/master/CHANGELOG.md

Not sure how replacing that will work, though.

**Comment by ndowens** [comment]
> Can you update to 4.8.4? At least there won't be new releases in this branch: https://github.com/wwmm/pulseeffects/blob/master/CHANGELOG.md
> 
> Not sure how replacing that will work, though.

Interesting, going to pipewrire too :)

**Pull request merged**